docs/api.md

```diff
@@ -95,11 +97,20 @@ See also [Actions](/docs/actions.md).
 
 #### ActionResult
 
-A partial state or [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to a partial state.
+A partial state, [thunk](#thunk) or [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to a partial state.
```

Comment:
Now `ActionResult` can be whatever you want as the `resolve` event allows you to transform it.
Or you are telling about `ActionResult` is the only thing the core will understand do trigger an update of the state ?